sdk/java-sdk/src/main/scala/com/akkaserverless/javasdk/impl/action/ActionsImpl.scala

```diff
@@ -134,10 +134,11 @@ private[javasdk] final class ActionsImpl(
           .recover { case NonFatal(ex) =>
             handleUnexpectedException(service, command, ex)
           }
-      case ErrorEffect(description, sideEffects) =>
+      case ErrorEffect(description, status, sideEffects) =>
         Future.successful(
           ActionResponse(
-            ActionResponse.Response.Failure(Failure(description = description)),
+            ActionResponse.Response.Failure(
+              Failure(description = description, grpcStatusCode = status.map(_.value()).getOrElse(0))),
```

Comment:
ok, `0` does not mean `SUCCESS` but 'unspecified' here, an will be converted to `UNKNOWN` in the proxy. Not super obvious but :+1: